pyproject.toml

```diff
@@ -78,7 +78,7 @@ pymssql = { version = "*", optional = true }
 pymysql = { version = "*", extras = ["rsa"], optional = true }
 neo4j = { version = "*", optional = true }
 opensearch-py = { version = "*", optional = true }
-cx_Oracle = { version = "*", optional = true }
+oracledb = { version = "*", optional = true }
```

Comment:
This is only needed during testing, so should be included as a dev dependency, not as part of the `extra`.